cms/envs/common.py

```diff
@@ -1127,3 +1130,12 @@
 
 # Partner support link for CMS footer
 PARTNER_SUPPORT_EMAIL = ''
+
+###################### Registration ##################################
+
+# For each of the fields, give one of the following values:
+# - 'required': to display the field, and make it mandatory
+# - 'optional': to display the field, and make it non-mandatory
+# - 'hidden': to not display the field
+
+REGISTRATION_EXTRA_FIELDS = {}
```

Comment:
It's required for the sake of this line: https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/user_api/views.py#L190

Now, we could exclude user_api routes from swagger if we don't want them exposed via Studio too. I'm open to suggestions.